embedded-workshop-book/src/hardware.md

```diff
@@ -106,10 +106,10 @@ $ ls /dev/tty.usbmodem*
 /dev/tty.usbmodem0006834208031
 ```
 
-The board has several switches to configure its behavior. The out of the box configuration is the one we want. If the above instructions didn't work for you, check the position of the on-board switches:
+The board has several switches to configure its behavior. The out of the box configuration is the one we want. If the above instructions didn't work for you, position the board so that the Button descriptions are horizontal and check the position of the on-board switches:
 
 - Switch SW6, on the top edge right corner, is set to the DEFAULT position; this is the right position of the two possible positions (nRF = DEFAULT). Note that this switch is protected by Kapton tape.
```

Comment:
On my board, SW7 is protected by Kapton tape, not this one.

Reply:
Good catch, on mine too (also not sure how one would protect a switch like that). Since @japaric is the original author of these instructions let's await his response

Reply:
ah, yes. The Kapton tape note should be on SW7. My bad, off-by-one error